README.md

```diff
@@ -48,7 +48,7 @@ This will produce something like this:
 
         TensorFlow version:  1.13.1
         nGraph bridge version: b'0.12.0-rc4'
```

Comment:
`nGraph bridge version: b'0.12.0-rc4'` should be changed to ` nGraph bridge version: b'0.12.0-rc5'`